resource-validator/src/main/scala/com/broadinstitute/dsp/resourceValidator/Main.scala

```diff
@@ -28,6 +28,9 @@ object Main
         val shouldCheckInitBuckets =
           Opts.flag("checkInitBuckets", "checks that init buckets for deleted runtimes are deleted").orFalse
 
+        val shouldCheckWorkers =
```

Comment:
I wonder if we would want to be more descriptive as in `shouldCheckDataprocWorkers`.